### ISSUE TITLE
Feature support for async-std

### ISSUE THREAD

**Issue by Roba1993** [scylladb/scylla-rust-driver#198]
For my project I would really like to use this library together with async-std.
Are you planning to support it over an cargo feature?

**Comment by psarna** [comment]
It's not on our roadmap, the initial design decision was to base the driver on tokio. Still, contributions are welcome :)

**Comment by Qix-** [comment]
This is the one thing that would prevent me from using Scylla. We use async-std, not Tokio. Async-std is more aligned with the standard library instead of reinventing everything itself. Could official support be added to the roadmap? It's not uncommon to have support for both in a single crate.

**Comment by cholcombe973** [comment]
So this may or may not be helpful to anyone but I started on a branch to introduce other executors: https://github.com/scylladb/scylla-rust-driver/compare/main...cholcombe973:scylla-rust-driver:main. I ran into a problem though in that Glommio doesn't support Send and scylla is using Arc all over the crate. I think it would be too much work for me to rip all that up but it might be helpful to someone else if they want to support async-std. I haven't used async-std so I don't know if it supports Send.
